docs/knowledgebase/solution-troubleshooting-etcd-failed-to-join-existing-cluster-error-message-during-peer-discovery.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "Solution - Troubleshooting etcd 'failed to join existing cluster' Error Message During Peer Discovery"
+linkTitle: "Solution - Troubleshooting etcd 'failed to join existing cluster' Error Message During Peer Discovery"
+---
+
+## Issue
+
+You are experiencing an issue where nodes cannot successfully join the cluster. Upon investigation, you also notice the following error messages in the logs:
+
+```bash
+# Truncated output...
+time="2018-09-24T13:40:20Z" level=info msg="not first cluster node, joining first node" action=create address=172.28.128.5 category=etcd host=node3 module=cp target=172.28.128.6
```

Comment:
This can no longer happen. This section can be removed. The join protocol in the CP doesn't work like this anymore.